spanner_dbapi/__init__.py

```diff
@@ -93,7 +93,7 @@ def connect(
     if not database.exists():
         raise ValueError("database '%s' does not exist." % database_id)
 
-    return Connection(database)
+    return Connection(database, instance)
```

Comment:
The `instance` reference can be accessed from within the [database](https://github.com/googleapis/python-spanner/blob/5fd4009dc8086e02b0deaa472f59958076c5a888/google/cloud/spanner_v1/database.py#L100).